doc/source/examples/dem-mp/heated-packed-bed/heated-packed-bed.rst

```diff
@@ -0,0 +1,372 @@
+
+    subsection lagrangian physical properties
+      set g                        = 0.0, 0.0 , -9.81
+      set number of particle types = 1
+      subsection particle type 0
+        set size distribution type            = uniform
+        set diameter                          = 6.4e-3
+        set number of particles               = 8849
+        set density particles                 = 7747
+        set young modulus particles           = 50e6
+        set poisson ratio particles           = 0.29
+        set restitution coefficient particles = 0.8
+        set friction coefficient particles    = 0.7
+        set rolling friction particles        = 0.02
+        set real young modulus particles      = 200e9
+        set thermal conductivity particles    = 42
+        set specific heat particles           = 464
+        set microhardness particles           = 3e9
+        set surface slope particles           = 0.056
+        set surface roughness particles       = 19.e-9
+        set thermal accommodation particles   = 0.7
+      end
+      set young modulus wall           = 50e6
+      set poisson ratio wall           = 0.33
+      set restitution coefficient wall = 0.8
+      set friction coefficient wall    = 0.7
+      set rolling friction wall        = 0.02
+      set real young modulus wall      = 100e9
+      set thermal conductivity wall    = 250
+      set microhardness wall           = 1.8e9
+      set surface slope wall           = 0.056
+      set surface roughness wall       = 0.1e-9
+      set thermal accommodation wall   = 0.7
+      set thermal conductivity gas     = 0.027
+      set specific heat gas            = 1006
+      set dynamic viscosity gas        = 1.85e-5
+      set specific heats ratio gas     = 1
+      set molecular mean free path gas = 68.e-9
+    end
+
+
+Model Parameters
+~~~~~~~~~~~~~~~~
+
+For the first two stages, the model parameters are defined as:
+
+.. code-block:: text
+
+    subsection model parameters
+      subsection contact detection
+        set contact detection method                = dynamic
+        set dynamic contact search size coefficient = 0.9
+        set neighborhood threshold                  = 1.3
+      end
+      subsection load balancing
+        set load balance method = frequent
+        set frequency           = 100000
+      end
+      set particle particle contact force method = hertz_mindlin_limit_overlap
+      set rolling resistance torque method       = constant_resistance
+      set particle wall contact force method     = nonlinear
+      set integration method                     = velocity_verlet
+      set solver type                            = dem_mp
+    end
+
+For the heating of the particles, the parameter ``disable position integration`` is set to ``true`` to freeze the position of the particles. This allows to use a higher time step for the evolution of the temperature. As particles are not moving, ``load balancing`` is no longer necessary.
```

Comment:
```suggestion
For the heating of the particles, the parameter ``disable position integration`` is set to ``true`` to freeze the position of the particles by disabling the time-integration of the particle velocity and position. This allows to use a higher time step for the evolution of the temperature since the collisions are not integrated in time anymore. As particles are not moving, ``load balancing`` is no longer necessary.
```